### ISSUE TITLE
can't view the tree

### ISSUE THREAD

**Issue by lydia1201** [nselem/evomining#3]
I have reached the last step, but when I click check tree,  it shows software error: No such file or directory at /var/www/html/EvoMining/cgi-bin/color_tree.pl line 629.

**Comment by nselem** [comment]
Hello Lydia
Could be that if you run it twice the trees some times overscript the files, or that it truly does not align the sequences due to the divergence. Does the aligned file contain data?

**Comment by code-chai** [comment]
Hi @nselem, I'm having a similar problem while trying to run EvoMining with default data. When I submit the results from the NP recruitment table for phylogenetic reconstruction, I receive the following message:
```
/var/www/html/EvoMining/exchange/ALL_curado.fasta_MiBIG_DB.faa_los17/blast/1.aln, No such file or directory at /var/www/html/EvoMining/cgi-bin/align_shave_tree.pl line 373.
```
Is `entradaAlignnn` the aligned file you mentioned in your previous comment? If so, that file is empty. 

Thank you for providing very detailed tutorial and instructions, I hope this problem can be solved so I can EvoMine to the fullest! 
Many thanks.

**Comment by nselem** [comment]
Hello code-chai
Did the test data run, is this a problem with your data? If you want, I always have time for a zoom call, and then if we succeed you can post here the answer and help other users

**Comment by code-chai** [comment]
Hi nselem, thank you so much for getting back to me! 
Unfortunately, I've tried pulling the docker image again and keep running into the same error when working with the example data and default options. 
This is the stage of failure:
![image](https://user-images.githubusercontent.com/91019705/145412180-47510feb-f219-4462-84b6-872dc31ca355.png)
Meanwhile in the terminal:
![image](https://user-images.githubusercontent.com/91019705/145416198-06a7eac4-a90d-40ed-b4cb-3286cb45cc1b.png)
align_shave_tree.pl is definitely there, but the file it may be trying to open on line 373 might not be. It would be great to go over it together, perl is a mystery to me!

**Comment by nselem** [comment]
Please send me an email and lets try to solve it. nselem@matmor.unam.mx

**Comment by code-chai** [comment]
After problem solving with @nselem, this error has been resolved!
\
Failure at the aligning stage prompted us to check the aligner, `muscle`, which can be found in the `/opt/` folder. While it had been installed successfully, it produced a `segmentation fault` when run. 
As discussed in the following forum post, this error is known to occur when running `muscle` on Ubuntu 20.04 on WSL2, which was my setup. (Windows 10 laptop running Ubuntu 20.04 via WSL2)
> https://www.biostars.org/p/9488765/#9489040 

After deleting the `muscle` folder, we followed 5heikki's guidance to install it again from source:
```
mkdir muscle
cd muscle
wget https://www.drive5.com/muscle/muscle_src_3.8.1551.tar.gz
tar xzvf muscle_src_3.8.1551.tar.gz
make ./muscle
```  
`muscle` was now working, so too was `startEvoMining.pl`, and alignment was successful.
\
Many thanks again to @nselem for taking the time to help me get EvoMining working! I hope to put this tool to good use.
